From b6534e54423a6f3e904c583072d55be0940e0e04 Mon Sep 17 00:00:00 2001
From: Rahul T R <r-ravikumar@ti.com>
Date: Tue, 18 Jul 2023 13:16:32 +0530
Subject: [PATCH] HACK: arm64: dts: ti: k3-j721s2-main: Disable sram for vpu

In psdk RTOS MSMC ram is dedicated for TIDL, So diabling it
for vpu

Signed-off-by: Rahul T R <r-ravikumar@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j721s2-main.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j721s2-main.dtsi b/arch/arm64/boot/dts/ti/k3-j721s2-main.dtsi
index 43dfe35c1bae..e715c61f3df9 100644
--- a/arch/arm64/boot/dts/ti/k3-j721s2-main.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-j721s2-main.dtsi
@@ -473,7 +473,7 @@
 		clock-names = "vcodec";
 		power-domains = <&k3_pds 179 TI_SCI_PD_EXCLUSIVE>;
 		sram=<&msmc_ram>;
-		sram-size=<0x22800>;
+		sram-size=<0x0>;
 	};
 
 	main_sdhci0: mmc@4f80000 {
-- 
2.17.1

